nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -646,6 +646,10 @@
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+## NixOS Wiki {#sec-release-25.05-wiki}
```

Comment:
Could also go in the Nixpkgs notes to be shared between both, but not sure if want that

Reply:
NixOS wiki being about NixOS sounds right to me